packages/env/lib/build-docker-compose-config.js

```diff
@@ -176,26 +176,41 @@ module.exports = function buildDockerComposeConfig( config ) {
 				image: 'mariadb',
 				ports: [ '3306' ],
 				environment: {
-					MYSQL_ALLOW_EMPTY_PASSWORD: 'yes',
+					MYSQL_ROOT_PASSWORD: 'password',
+					MYSQL_DATABASE: 'wordpress',
```

Comment:
Setting the database here, makes sure the container creates it (which was done previously in the wordpress image)

Reply:
This is also why we need a separate container for the database of the tests env.